### ISSUE TITLE
fix design inconsistencies

### ISSUE THREAD

**Issue by open-contracts** [open-contracts/open-contracts.github.io#6]
Edit: making this about the remaining inconsistencies in the layouts. See post below.

**Comment by open-contracts** [comment]
I like where this is going, but we still need to iterate. Before optimizing the prompt boxes (which still need work), let's try this:

![image](https://user-images.githubusercontent.com/90533597/140794151-f30e7dad-3e85-4511-a6b5-53c1d8c43275.png)

Meaning, let's separate the site background from the grid-view and function-view tab background. Make the site background fully white. Then we can try out different colors for the tabs. I'd like to try a cool (w/o the current yellow tint) grey, like in the picture, and with no border line.

**Comment by l-monninger** [comment]
@Jonas-Metzger When you say no border, are you just referring to the sides, or do you not want the hr too?

**Comment by open-contracts** [comment]
I meant also the top, to be consistent. but looking at it now, having a border everywhere probably looks better. So add the side (and bottom) borders as well please.



Question. Instead of the bottom borders, I was also wondering about the following: what if the grey tab always extends all the way until the bottom of the page?
![image](https://user-images.githubusercontent.com/40152568/141836487-4493c22f-b25a-4c11-b558-589a66b6863d.png)
Also for the function grid view: 
![image](https://user-images.githubusercontent.com/40152568/141836970-5dcb4059-ecf4-4f1e-849f-d8d3852e6a92.png)
Do you think that would look better?

I originally thought all the way to the bottom looks better, but now I kinda like the current, compact layout (assuming we properly draw a   box around it)

**Comment by open-contracts** [comment]
Proportions/layouts in the log are sometimes inconsistent. I'll mention different things that bug me below, but they're all related so let's keep the discussion in this feed:

1.) For 'description' and 'input', can you use the same (slightly smaller) font size that is also used in the log prompts? The result prompt has a different layout as well https://github.com/open-contracts/open-contracts.github.io/issues/32
![image](https://user-images.githubusercontent.com/90533597/142063677-4ad728a0-ce95-4b33-9760-806b53cca45c.png)

2.) The prompt titles all start at different indentations. Can you force them to start at the same position, even if the icons may not be exactly the same width?

3.) The description icon is quite large and heavy compared to the other icons. Consider using the slightly smaller and lighter one from the warning at the top:
![image](https://user-images.githubusercontent.com/90533597/142064260-31565bbe-819c-43bd-94a8-7c39fa5bfc51.png)

4.) Try harmonizing the "x" icon between the top input and enclave input prompts. I'd try having both in bold (fits better with f_x), and at an intermediate size between the current two (should be more in line with the other icons). 

5.) The gaps between content and box are a bit all over the place. (see first picture of this post). Try using the top/bottom gap size of the Input and description boxes (which seems to be the same) as the horizonal gap size around their content (=the description text and input list), but leaving the prompt titles and icons as they are. It's still a different gap than the enclave prompts, but I think that might be more harmonic bc their text starts further inside as well...

**Comment by open-contracts** [comment]
![image](https://user-images.githubusercontent.com/90533597/142103213-952e97a2-202b-4b54-af6f-14e7afac7e91.png)
We should also remove the _'oracleInput'_ text. It doesn't really contain information and looks like its part of the message.

**Comment by open-contracts** [comment]
The previous post can mostly be summarized by the following:

Can we just have one abstract layout for elements in the function tab (including input, description, log promts such as error, result, user input), which defines the positions of

```
      ┌───────────────────────┐
      │                       │
      │  ┌────┐  ┌─────────┐  │
      │  │ICON│  │TITLE    │  │
      │  ├────┴──┴─────────┤  │
      │  │CONTENT          │  │
      │  │                 │  │
      │  └─────────────────┘  │
      │                       │
      └───────────────────────┘
```

ands forces icon, title, and content (whether its just text, or an input field, or multiple input fields) to start at the same indentation and use the same respective fonts across different elements?

**Comment by l-monninger** [comment]
@Jonas-Metzger So, there actually is just one standard layout for the prompts. I'd just overwritten it in various ways throughout our back-and-forths.

**Comment by l-monninger** [comment]
@Jonas-Metzger  Also, how long does it take you to type those ascii graphs haha?

**Comment by open-contracts** [comment]
awesome, getting much cleaner!

can we move the notification 'copy' button to the right (to make the layout more in line with the 'submit' button in the input prompts) and make sure the button always has the same size (see bug below)

![image](https://user-images.githubusercontent.com/90533597/142742675-3e783f59-3d6b-41cd-bb31-f50a6ae3b4d0.png)


Currently the gaps between the log elements are wider than those on their sides. Same thing for the function grid:

![image](https://user-images.githubusercontent.com/90533597/142742751-45051fdd-3373-4703-ab4a-d0c0de1f5472.png)

It would be nice if we used the wider gap size for the side borders as well. 

I now think it might look nicer if we also increased the border-gap size inside the log elements (sides and top/bottom). I would at least double it:
![image](https://user-images.githubusercontent.com/90533597/142742811-c04450ad-7ebb-4064-9698-b94461752308.png)
